packages/stencil/components/input/src/components/osds-input/osds-input.e2e.screenshot.ts

```diff
@@ -38,48 +38,202 @@ describe('e2e:osds-input', () => {
 
   const screenshotActions = [
     {
+      // Will display the default input
       actionDescription: 'no action',
       action: () => {
-        // noop
+        // nothing
       },
     }, {
+      // Will display the input with an eye icon to show/hide the value
+      actionDescription: 'hideable',
+      action: () => el.setProperty('hideable', true),
+    }, {
+      // Will display the input with a cross icon to clear the input
+      actionDescription: 'clearable',
+      action: () => el.setProperty('clearable', true),
+    }, {
+      // Will contrast the colors of the input
       actionDescription: 'contrasted',
       action: () => el.setProperty('contrasted', true),
     }, {
-      actionDescription: 'disabled',
```

Comment:
No more test when component is disabled? Disabled only tested with a spinner or with contrasted colours?